<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>-1</y>
     <width>591</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Open The File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>File Location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Split The File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>CSS Parts</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Open Explorer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Merge Files Together</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Delete The Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Save File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="Tab">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Background</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>471</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>B:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>G:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>R:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QSlider" name="BlueSlider_2">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="pageStep">
                  <number>5</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBothSides</enum>
                 </property>
                 <property name="tickInterval">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="GreenSlider_2">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="pageStep">
                  <number>5</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBothSides</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="RedSlider_2">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="pageStep">
                  <number>5</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBothSides</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Hex:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="text">
                <string>#000000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Convert to RGB</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string>Foreground</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>471</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_19">
                 <item>
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>B:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_20">
                 <item>
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>G:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_21">
                 <item>
                  <widget class="QLabel" name="label_17">
                   <property name="text">
                    <string>R:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_18">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QSlider" name="BlueSlider_3">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="pageStep">
                  <number>5</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBothSides</enum>
                 </property>
                 <property name="tickInterval">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="GreenSlider_3">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="pageStep">
                  <number>5</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBothSides</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="RedSlider_3">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="pageStep">
                  <number>5</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBothSides</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Hex:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="text">
                <string>#000000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_8">
               <property name="text">
                <string>Convert to RGB</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Test</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Padding:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Margin:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QSlider" name="PaddingSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="MarginSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PaddingSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_12</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MarginSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RedSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GreenSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BlueSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BlueSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_14</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GreenSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_16</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RedSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_18</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
